### ISSUE TITLE
No Trilinos checkin emails

### ISSUE THREAD

**Issue by jhux2** [trilinos/Trilinos#853]
@bmpersc @jwillenbring @trilinos/framework I'm not receiving any of the Trilinos checkin emails.  The last ones I received are around November 11th.  I haven't changed any settings in my profile.  Could there a problem upstream somewhere?

**Comment by bmpersc** [comment]
@jhux2 I have checked and I am seeing some similar behavior, the last checkin message I got was from 11/11/2016. I can look at some settings, but I'm not sure I'll be able to do anything about this. If this is that github has changed something on their servers we will have no way to tell that or would be able to fix it.

Just out of curiousity, are you a developer on another github project? If so is that project showing this same behavior?

**Comment by jhux2** [comment]
@bmpersc I'm part of NaluCFD, but I've never received those checkin emails, so I don't know whether that project.  I sure hope github didn't silently break/stop email checkin notifications.

**Comment by jwillenbring** [comment]
I looked on the server that hosts the checkins mail list and didn't notice anything unusual, other than the fact that there were no messages in the archive since 11/11 (so Sandia didn't start filtering the messages).

@maherou did you happen to modify any settings for the checkins mail list recently?

**Comment by jhux2** [comment]
@jwillenbring Any updates on this?

**Comment by jwillenbring** [comment]
After initially implementing a workaround for this, I contacted GitHub to see if they had received a lot of bounces, and that caused the emails to be disabled. That was the case. GitHub made a change on their end, and I was able to redirect the messages back to the regular checkins list.

In the future, I need to remember that this is a possibility. Both John P and I spent a lot of time investigating this and Brent P and Navisite also spent time on it, and in the end, after almost all other possibilities were ruled out, I finally contacted GitHub, and they were able to fix it very quickly. I thought of this possibility very early on, but didn't think it was likely the issue, so I delayed contacting GitHub.
